.github/workflows/publish_to_wiki.yml

```diff
@@ -0,0 +1,55 @@
+name: Publish to Wiki
+
+# Trigger this action only if there are changes pushed to the docs/** directory under the main branch
+on:
+  push:
+    paths:
+      - markdown/wiki/** # This includes all sub folders
+    branches:
+      - main # This can be changed to any branch of your preference
+
+env:
+  USER_TOKEN: ${{ secrets.WIKI_ACTION_TOKEN }} # This is the repository secret
+  USER_NAME: <USERNAME> # Enter the username of your (bot) account
+  USER_EMAIL: <EMAIL> # Enter the e-mail of your (bot) account
```

Comment:
Env. vars leak like air through a sieve.  Would you consider specifying the secret values directly/only in the places where they're needed instead of globally?